utilities/mtv_migration.py

```
@@ -58,6 +61,15 @@ def migrate_vms(
             vm["id"] = vm_data["id"]
             LOGGER.info(f"VM '{vm_name}' -> ID '{vm['id']}'")
 
+    # Check MTV version support for target_affinity
+    if not is_mtv_version_supported(ocp_admin_client, "2.10.0"):
+        LOGGER.warning("targetAffinity requires MTV 2.10.0+, skipping affinity configuration")
+        target_affinity = None
+    else:
+        target_affinity = plan.get("target_affinity")
+        if target_affinity:
+            LOGGER.info(f"Using target_affinity: {target_affinity}")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)